packages/css/src/ordered-list/README.md

```diff
@@ -0,0 +1,15 @@
+# Ordered List
+
+## Introductie
```

Comment:
This differs from the [component docs guidelines](https://github.com/Amsterdam/design-system/blob/main/documentation/component-docs-guidelines.md), the docs shouldn't have an 'Introductie' heading and 'Specificaties' should be 'Richtlijnen' I think